tsconfig.base.json

```diff
@@ -94,21 +94,18 @@
       "@energinet-datahub/dh/market-participant/actors/feature-organizations": [
         "libs/dh/market-participant/actors/feature-organizations/index.ts"
       ],
-      "@energinet-datahub/dh/market-participant/actors/shell": [
-        "libs/dh/market-participant/actors/shell/src/index.ts"
-      ],
       "@energinet-datahub/dh/market-participant/data-access-api": [
         "libs/dh/market-participant/data-access-api/src/index.ts"
       ],
       "@energinet-datahub/dh/market-participant/domain": [
         "libs/dh/market-participant/domain/index.ts"
       ],
+      "@energinet-datahub/dh/market-participant/shared-ui": [
+        "libs/dh/market-participant/shared-ui/index.ts"
+      ],
```

Comment:
suggestion: the naming pattern is usually `<lib-type>-<lib-name>`. So `ui-shared`.

Reply:
Fixed - still thinks it sounds better the other way around :-)